example/src/MapViewExample.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { MapView, MapMarker, MapCallout, withTheme } from "@draftbit/ui";
+import { Dimensions, StyleSheet, Text, View } from "react-native";
+
+const MapViewExample = ({ theme }) => (
+  <View style={styles.container}>
+    <MapView
+      showsCompass
+      style={styles.map}
+      apiKey="myApiKey"
+      latitudeDelta={0.0922}
+      longitudeDelta={0.0421}
+      latitude={40.741895}
+      longitude={-73.989308}
+    >
+      <MapMarker
+        latitude={40.741895}
+        longitude={-73.989308}
+        pinColor={theme.colors.primary}
+        title="Draftbit"
+        description="A simple MapView example"
+      />
+      <MapMarker
+        latitude={40.741895}
+        longitude={-73.979308}
+        pinColor={theme.colors.secondary}
+      >
+        <MapCallout showTooltip>
```

Comment:
On the Web, Tooltip is covering the Marker. It should be shifted above the marker. 
![image](https://user-images.githubusercontent.com/1259534/127867594-3d6af3d4-fd6f-45ed-a4c7-b793c965a576.png)